app/components/forms/discount-codes/create-discount-codes-form.js

```diff
@@ -0,0 +1,61 @@
+import Ember from 'ember';
+import FormMixin from 'open-event-frontend/mixins/form';
+
+const { Component } = Ember;
+
+export default Component.extend(FormMixin, {
+  getValidationRules() {
+    return {
+      inline : true,
+      delay  : false,
+      on     : 'blur',
+
+      fields: {
+        discountCode: {
+          identifier : 'discount_code',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Please enter a discount code')
+            }
+          ]
+        },
+        numberOfAccessTickets: {
+          identifier : 'number_of_discount_tickets',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Please enter the number of tickets')
+            }
+          ]
+        },
+        discountAmount: {
+          identifier : 'discount_amount',
+          rules      : [
+            {
+              type   : 'integer',
+              prompt : this.l10n.t('Please enter the number of tickets')
+            }
+          ]
+        },
+        discountPercent: {
+          identifier : 'discount_percent',
+          rules      : [
+            {
+              type   : 'integer',
```

Comment:
Percentage can be a floating value.
![screenshot from 2017-06-24 11-46-47](https://user-images.githubusercontent.com/12807846/27506304-1320ed74-58d3-11e7-8ff8-ad41269c97dc.png)